deployment/kubernetes/helm/pulsar/values.yaml

```diff
@@ -0,0 +1,388 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+## Namespace to deploy pulsar
+namespace: pulsar
+namespaceCreate: yes
+
+## which extra components to deploy
+extra:
+  # Pulsar proxy
+  proxy: yes
+  # Bookkeeper auto-recovery
+  autoRecovery: yes
+  # Pulsar dashboard
+  dashboard: yes
+  # Bastion pod for administrative commands
+  bastion: yes
+  # Monitoring stack (prometheus and grafana)
+  monitoring: yes
+
+## Which pulsar image to use
+image:
+  repository: apachepulsar/pulsar
+  tag: latest
+  pullPolicy: IfNotPresent
+
+## Pulsar: Zookeeper cluster
+## templates/zookeeper-statefulset.yaml
+##
+zookeeper:
+  component: zookeeper
+  replicaCount: 3
+  updateStrategy:
+    type: OnDelete
+  podManagementPolicy: OrderedReady
+  nodeSelector:
+    cloud.google.com/gke-nodepool: default-pool
```

Comment:
some of these values are very gke specific. Does it make sense to have the values.yaml provide defaults that can be shared across platforms and then add a gke-values.yaml for overriding specific ones?